src/DAL/repositories/jobRepository.ts

```diff
@@ -106,13 +107,14 @@ export class JobRepository extends GeneralRepository<JobEntity> {
     }
   }
 
-  public async getJob(id: string, shouldReturnTasks = true): Promise<IGetJobResponse | undefined> {
+  public async getJob(id: string, query: IJobsQuery): Promise<IGetJobResponse | undefined> {
```

Comment:
this will change the behavior of the method as till now the "shouldReturnTasks" was true.
Please make sure this default behavior is being kept

Reply:
I already made sure